### DISCUSSION TITLE
How to remove the transition of color when dragging?

### DISCUSSION THREAD

**Discussion by zed-wong** [saadeghi/daisyui#1454] [Q&A] [answered]
When I drag an element of a list, the background color changes. How can I remove this?

![Screenshot from 2022-12-21 23-54-41](https://user-images.githubusercontent.com/36271125/208934649-a3bd3398-4e34-4732-976c-326fa972dd45.png)

Also, when I click on it, the text on the right side would flash, how can I remove it?

![Screenshot from 2022-12-21 23-57-47](https://user-images.githubusercontent.com/36271125/208935011-aafac303-8413-4e00-aabf-3c364b751ff7.png)

This screenshot was captured right after I clicked on it. As you can see the `0` in the first image disappeared.

**Comment by saadeghi** [comment, marked as answer]
If you don't want the `active` color of the menu items, you can either change them in CSS like this:  
https://play.tailwindcss.com/AB8OuJcLiE?file=css
```css
.menu > li > *:active {
  @apply bg-base-300 text-inherit;
}
```
Or using utility classes like this:  
https://play.tailwindcss.com/xMuTvYyQvY
```html
<ul class="menu [&>li>*:active]:bg-base-300 [&>li>*:active]:text-inherit">
```

I can't help you about the color of that `0` unless you give me a code example. By default, menu items use the `currentColor` for text so it will be whatever color is coming from their parent. So maybe adding a text color for it will can help. like `text-base-content`

**Reply by zed-wong** [reply]
Thanks! After applying this, both problems are solved!
```
.menu > li > *:active {
  @apply bg-base-300 text-inherit;
}
```
